views/help/howtos.pug

```diff
@@ -0,0 +1,36 @@
+extends ../layouts/help-layout
+
+block content
+
+  .container
+    h1 Maximum Help Topics
+
+    ul.nav.nav-tabs
+      li.active
+        a(data-toggle='tab', href='#howto') How-to
+      li
+        a(data-toggle='tab', href='#commands') Commands
+      li
+        a(data-toggle='tab', href='#support') Support
+    .tab-content
```

Comment:
Please add a new line between logical blocks (i.e. between `.tab-content` and previous block, `ul.nav.nav-tabs`